src/Utilities/Workspaces/SyntaxNodeExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Simplification;
+
+namespace Analyzer.Utilities
+{
+    internal static class SyntaxNodeExtensions
+    {
+        private static Optional<SyntaxAnnotation?> s_addImportsAnnotation;
+
+        private static SyntaxAnnotation? AddImportsAnnotation
+        {
+            get
+            {
+                if (!s_addImportsAnnotation.HasValue)
+                {
+                    var property = typeof(Simplifier).GetTypeInfo().GetDeclaredProperty("AddImportsAnnotation");
+                    if (property is object)
+                    {
+                        s_addImportsAnnotation = property.GetValue(null) as SyntaxAnnotation;
```

Comment:
You could merge the if and else by calling `property?.GetValue`.

Reply:
This type is added as part of #3230. That one will likely merge before this is marked ready for review, in which case it will no longer be part of the diff.